src/main/webapp/WEB-INF/index.yaml

```diff
@@ -0,0 +1,7 @@
+indexes:
+
```

Comment:
optional: add a short comment explaining where this is used. Eg: "This index need when querying ... etc".

Reply:
Done!